index.js

```diff
@@ -1,18 +1,33 @@
 /***** Deliverable 1 *****/
+
 const header = document.querySelector("#header")
 console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = "blue"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+document.getElementsByTagName('img')[0].src = traveler.photo
+document.getElementsByTagName('img')[0].alt = "Raffy pic"
+document.getElementsByTagName('h2')[0].innerText = traveler.name
+document.getElementsByTagName('em')[0].innerText = traveler.nickname
+console.log('This is what the traveler object looks like: ', traveler)
 
 
 /***** Deliverable 4 *****/
-
+const ul = document.querySelector('ul#animals')
+traveler.animalSightings.forEach(function (sighting) {
+    const li = document.createElement('li')
+    li.setAttribute('data-id', `${sighting.id}`)
```

Comment:
This works!
- Another solution as you may have read about in the article I linked you to would be:
```js
li.dataset.id = sighting.id
```

You'll also be able to access this property by either writing `li.getAttribute('data-id')` or `li.dataset`